### DISCUSSION TITLE
Disabling the new "Password login alert" email

### DISCUSSION THREAD

**Discussion by orenaksakal** [pocketbase/pocketbase#5104] [Q&A] [answered]
Is there a way to opt-out from this automated email? It is not making a lot of sense if user has both oauth and password login assigned as its sent every time they login. It makes more sense to send it only once or have option to disable/change the email template in admin panel

**Comment by ganigeorgiev** [comment, marked as answer]
> It is not making a lot of sense if user has both oauth and password login assigned as its sent every time they login. 

It is sent **only** if the user authenticate with password **and if they have OAuth2 linked**.

In most cases the OAuth2 is usually used as "upgrade" to the password login, so most users I imagine will use one or the other.

Additionally it is currently send only once per 7 days, aka. if the user authenticate with password multiple times within 7 days they will receive only 1 email. The counter resets if the user perform OAuth2 -> Password auth sequence.

I'm OK with adjusting the frequency but let's wait to see what other users think.

Note that this alert is just a temp workaround and will be replaced with the refactoring in favor of "Login from new device" emails.

**Reply by ganigeorgiev** [reply]
@IsmaelAlvarez You are commenting to an old thread that already has been resolved with v0.23+ where option to disable the login alerts exist _(also note that the above and no longer available OAuth2 alerts are different from the generic login alert available in v0.23+)_:
![options_screenshot](https://i.imgur.com/nDiIZzl.png)

_(aka. disable the "Send email alert for new logins" toggle)_

Note that each auth collection (e.g. users, _superusers) has its own options and you'll have to toggle it off individually.

_edit: it seems that you've deleted your post but the above remains valid_
